src/lightning_app/cli/lightning_cli.py

```diff
@@ -26,13 +34,149 @@ def get_app_url(runtime_type: RuntimeType, *args) -> str:
         return "http://127.0.0.1:7501/admin" if LOCAL_LAUNCH_ADMIN_VIEW else "http://127.0.0.1:7501/view"
 
 
+@click.group()
+def clusters():
+    """Manage your Lightning.ai compute clusters.
+
+    Lightning.ai supports running apps on your own cloud provider account. This feature is currently in preview and
+    requires an extra activation step. Reach out to support@lightning.ai to enable this feature.
+    """
+    pass
+
+
+@clusters.command("create")
+@click.argument("cluster_name", callback=_check_cluster_name_is_valid)
+@click.option("--provider", "provider", type=str, default="aws", help="cloud provider to be used for your cluster")
+@click.option("--external-id", "external_id", type=str, required=True)
+@click.option(
+    "--role-arn", "role_arn", type=str, required=True, help="AWS role ARN attached to`the associated resources."
+)
+@click.option(
+    "--region",
+    "region",
+    type=str,
+    required=False,
+    default="us-east-1",
+    help="AWS region which is used to host the associated resources.",
+)
+@click.option(
+    "--instance-types",
+    "instance_types",
+    type=str,
+    required=False,
+    default=",".join(default_instance_types),
+    help="Instance types which you desire to support for computer jobs within the cluster.",
+)
+@click.option(
+    "--cost-savings",
+    "cost_savings",
+    type=bool,
+    required=False,
+    default=False,
+    is_flag=True,
+    help="using this flag ensures that the cluster is created with a profile that is optimized for "
+    "cost saving, making runs cheaper but start-up times may increase",
+)
+@click.option(
+    "--edit-before-creation",
+    default=False,
+    is_flag=True,
+    help="Edit the created cluster spec before submitting to API server.",
+)
+@click.option(
+    "--wait",
+    "wait",
+    type=bool,
+    required=False,
+    default=False,
+    is_flag=True,
+    help="using this flag CLI will wait until the cluster is running",
+)
+def create_cluster(
+    cluster_name: str,
+    region: str,
+    role_arn: str,
+    external_id: str,
+    provider: str,
+    instance_types: str,
+    edit_before_creation: bool,
+    cost_savings: bool,
+    wait: bool,
+    **kwargs,
+):
+    """Create a Lightning.ai compute cluster with provided cloud provider credentials."""
+    if provider != "aws":
+        click.echo("only provider aws is supported today")
```

Comment:
Please use: 

Only AWS is supported for now. But support for more providers is coming soon.